client/gdc/maf.js

```diff
@@ -304,7 +304,7 @@ async function getFilesAndShowTable(obj) {
 	for (const f of result.files) {
 		const row = [
 			{
-				html: `<a href=https://portal.gdc.cancer.gov/cases/${f.case_uuid} target=_blank>${f.case_submitter_id}</a>`,
+				html: `<a href=https://portal.gdc.cancer.gov/cases/${f.case_uuid} target=_blank onclick="event.stopPropagation()">${f.case_submitter_id}</a>`,
```

Comment:
if this cannot be implemented in table.ts, should make a note there that clickable link in html must have ...